.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules
 coverage
 .#*
 docs/_site
-docs/files
\ No newline at end of file
+docs/files
+.idea
```

Comment:
This too seems unrelated to the new Node types. There is a non-versioned gitignore file located at _.git/info/exclude_ if you ever have the need to ignore files that your text editor creates ([see here](https://help.github.com/articles/ignoring-files/), also, for how to set up global gitignores on your machine).